### PR TITLE
[#2034] Improvement (trino-connector): Add datatype , catalog test cases for the Trino connector

### PR DESCRIPTION
**Pull request by diqiu50**
### What changes were proposed in this pull request?

Add datatype , catalog test cases for the Trino connector

### Why are the changes needed?

Issue: #2034

### Does this PR introduce _any_ user-facing change?

NO

### How was this patch tested?

TriniQueryIT

### CODE REVIEW

**Comment by jerryshao** [comment]
@yuqi1129 can you please review this?

File: integration-test/src/test/resources/trino-ci-testset/testsets/hive/00006_datatype.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+   serde_lib = 'org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe',
+   serde_name = 'tb01',
+   table_type = 'MANAGED_TABLE'
+)"
+
+INSERT: 1 row
+
+INSERT: 1 row
+
+"Sample text 1","Text1               ","65","123.456","7.89","12.34","false","1","100","1000","1000","100000","2024-01-01","2024-01-01 08:00:00.000"
```
Comment (yuqi1129) [inline integration-test/src/test/resources/trino-ci-testset/testsets/hive/00006_datatype.txt:37]: A question about space in value: Do you write it deliberately?
image

**Comment by yuqi1129** [comment]
Others seem to be okay with me.

File: integration-test/src/test/resources/trino-ci-testset/testsets/hive/00006_datatype.txt (hunk 1 above)
Comment (diqiu50) [reply integration-test/src/test/resources/trino-ci-testset/testsets/hive/00006_datatype.txt:37]: The field f2 is defined as `CHAR(20)`, so it is filled with spaces if the input is shorter than 20 characters.”

**Review by yuqi1129 — APPROVED** [review]
LGTM

**Pull request merged**